_data/simple-icons.json

```
@@ -761,6 +761,11 @@
             "hex": "7D929E",
             "source": "https://www.aseprite.org/"
         },
+        {
+            "title": "Ask Ubuntu",
+            "hex": "DC461D",
+            "source": "https://askubuntu.com/"
```

Could you add the https://stackoverflow.com/legal/trademark-guidance as the `guidelines` entry as it was added for the rest of the StackExchange icons in #5164?

```suggestion
            "source": "https://askubuntu.com/",
            "guidelines": "https://stackoverflow.com/legal/trademark-guidance"
```